generated/nidaqmx/nidaqmx_service.cpp

```diff
@@ -30,7 +30,9 @@ namespace nidaqmx_grpc {
       NiDAQmxLibraryInterface* library,
       ResourceRepositorySharedPtr session_repository, 
       const NiDAQmxFeatureToggles& feature_toggles)
-      : library_(library), session_repository_(session_repository), feature_toggles_(feature_toggles)
```

Comment:
Splitting these to separate lines just makes it easier to conditionally codegen additional arguments